LondonBritishLibrary/orient/BLorient586.xml

```diff
@@ -54,41 +84,231 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <locus from="73r" to="195v"/>
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                </layout>
                             </layoutDesc>
                         </objectDesc>
                         
                         <handDesc>
                             
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is written in a small hand of the <date>18th century</date>
-                                </desc>
-                                
-                                <date notBefore="1700" notAfter="1800"/>
-                                
+                                <desc>Written in a small hand.</desc>
+                                <date notBefore="1700" notAfter="1799"/>
+                                <persName ref="PRS13814MekhaG" role="scribe"/>
                             </handNote>
-                            
                         </handDesc>   
+                        
                         <additions>
-                            
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote">The note states that the manuscript belongs to the church of 
+                                        <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                                    <q xml:lang="gez">ምዕራፍ፡ ዝማሬ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a2">
+                                    <locus target="#111r"/>
+                                    <desc type="GuestText"></desc>
+                                    <q xml:lang="gez"><seg part="I">ምህላ፡ ፱ ዘምሉእ፡ ፫ እምኔሆሙ፡ ምህላ፡ ዘሆሣዕና፡ ዘዓርብ፡
+                                    ወምህላ፡ በበዓለ፡ ፶ ዘረቡዕ።</seg></q>
+                                </item>
+                                
+                                <item xml:id="a3">
+                                    <locus target="#111v"/>
+                                    <desc type="GuestText"></desc>
+                                    <q xml:lang="gez"><seg part="I">ወግ፡ ዘፋሲካ፡ እምፋሲካ፡ እስከ፡ ዕርገት፡ ዘእኁድ፡ ትንሣኤ፡ ዮም፡ ፋሲካ፡ ይትበሀል። እምፋሲካ፡ እስከ፡
```

Comment:
This seems to be a text called The Order of the Easter, maybe we can make an ID for that?

Reply:
It is difficult for me to estimate. I found this nowhere, but I cannot tell whether it might be an excerpt from something.